sdk/storage/Azure.Storage.DataMovement.Blobs/README.md

```diff
@@ -302,13 +226,92 @@ destinationResource: blobs.FromContainer(
+
+Instantiate the BlobContainerClient
+```C# Snippet:ExtensionMethodCreateContainerClient
+BlobServiceClient service = new BlobServiceClient(serviceUri, credential);
+
+BlobContainerClient container = service.GetBlobContainerClient(containerName);
+```
+
+Upload a local directory to the root of the container
+```C# Snippet:ExtensionMethodSimpleUploadToRoot
+DataTransfer transfer = await container.StartUploadDirectoryAsync(localPath);
+
+await transfer.WaitForCompletionAsync();
+```
+
+Upload a local directory to a virtual directory in the container by specifying a directory prefix
+```C# Snippet:ExtensionMethodSimpleUploadToDirectoryPrefix
+DataTransfer transfer = await container.StartUploadDirectoryAsync(localPath, blobDirectoryPrefix);
+
+await transfer.WaitForCompletionAsync();
+```
+
+Upload a local directory to a virtual directory in the container specifying more advanced options
+```C# Snippet:ExtensionMethodSimpleUploadWithOptions
+BlobContainerClientTransferOptions options = new BlobContainerClientTransferOptions
+{
+    BlobContainerOptions = new BlobStorageResourceContainerOptions
+    {
+        BlobDirectoryPrefix = blobDirectoryPrefix
+    },
+    TransferOptions = new DataTransferOptions()
+    {
+        CreationPreference = StorageResourceCreationPreference.OverwriteIfExists,
+    }
+};
+
+DataTransfer transfer = await container.StartUploadDirectoryAsync(localPath, options);
+
+await transfer.WaitForCompletionAsync();
+```
+
+Download the entire container to a local directory
+```C# Snippet:ExtensionMethodSimpleDownloadContainer
+DataTransfer transfer = await container.StartDownloadToDirectoryAsync(localDirectoryPath);
+
+await transfer.WaitForCompletionAsync();
+```
+
+Download a directory in the container by specifying a directory prefix
+```C# Snippet:ExtensionMethodSimpleDownloadContainerDirectory
+DataTransfer tranfer = await container.StartDownloadToDirectoryAsync(localDirectoryPath2, blobDirectoryPrefix);
+
+await tranfer.WaitForCompletionAsync();
+```
+
+Download from the container specifying more advanced options
+```C# Snippet:ExtensionMethodSimpleDownloadContainerDirectoryWithOptions
+BlobContainerClientTransferOptions options = new BlobContainerClientTransferOptions
+{
+    BlobContainerOptions = new BlobStorageResourceContainerOptions
+    {
+        BlobDirectoryPrefix = blobDirectoryPrefix
+    },
+    TransferOptions = new DataTransferOptions()
+    {
+        CreationPreference = StorageResourceCreationPreference.OverwriteIfExists,
+    }
+};
+
+DataTransfer tranfer = await container.StartDownloadToDirectoryAsync(localDirectoryPath2, options);
+
+await tranfer.WaitForCompletionAsync();
+```
+
 ## Troubleshooting
 
-***TODO***
+See [Handling Failed Transfers](#handling-failed-transfers) and [Enabling Logging](https://learn.microsoft.com/en-us/dotnet/azure/sdk/logging) to assist with any troubleshooting.
 
 ## Next steps
 
-***TODO***
+Get started with our [Share Files Samples][share_samples].
```

Comment:
Did you intend for the Blob package to point at Share samples and vice versa?

Reply:
Yes, cause this is the Next Steps section. Just wanted to make sure if people wanted to dive into what else is there for DMlib they can